app/models/spree/line_item.rb

```diff
@@ -219,6 +219,12 @@ def unit_value
       final_weight_volume / quantity
     end
 
+    def unit_price_price_and_unit
```

Comment:
shouldn't it be `unit_price_and_unit`? I also think we should document why this is random. No other dev will understand.

Reply:
Actually, I havent' spend to much time on it since @andrewpbrett will re-do this part. 
I wanted to create a structure like:
````
def unit_price 
  price = ... whateever
  unit = ... whatever
  return {
    price: price, 
    unit: unit
  }
end
```

Reply:
well, we can live with it until he works on it :+1: